lib/core/app/controllers/file_fingerprints_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class FileFingerprintsController < ApplicationController
+  def index
+    render json: JSONAPI::ResourceSerializer.new(FileFingerprintResource).serialize_to_hash(resources)
```

Comment:
call the `json_resources` method in applicationControllerConcern rather than `JSONAPI::ResourceSerialize`